Jenkinsfile.deploy

```
@@ -159,6 +237,20 @@ pipeline {
             steps {
                 script {
                     launchParallelForEachImage("Test", {img -> testImage(img)})
+                    build_stages = [:]
+                    IMAGES.each { image -> build_stages[image] = {
+                            initWorkspace(image)
+                            dir(getWorkspacePath(image)) {
+                                try {
+                                    sh "make ${image} ignore_test=${shouldSkipTests()} cekit_option='--work-dir .'"
+                                } finally {
+                                    junit testResults: 'target/test/results/*.xml', allowEmptyResults: true
+                                }
+                            
+                            }
+                        }
+                    }
+                    parallel build_stages
                 }
```

@shashankcuber there was a problem in the rebase, those lines should deleted (just apply the suiggestion)

```suggestion
                script {
                    launchParallelForEachImage("Test", {img -> testImage(img)})
                }
```